components/eam/bld/namelist_files/use_cases/2010_scream_hr.xml

```diff
@@ -113,8 +113,12 @@
 <prescribed_aero_cycle_yr>01</prescribed_aero_cycle_yr>
 
 <!-- Do radiation every five minutes for ne1024 -->
-<iradsw hgrid="ne1024np4"> 4 </iradsw>
-<iradlw hgrid="ne1024np4"> 4 </iradlw>
+<iradsw hgrid="ne256np4"> 3 </iradsw>
```

Comment:
Actually, these should probably all go in namelist_defaults_eam.xml too now that I think about it; then we only have to change them in one place.

Reply:
Yeah, having less copying and pasting makes sense.